docs/readme_kr.md

```diff
@@ -10,7 +10,7 @@
     <a href="https://codecov.io/gh/anuraghazra/github-readme-stats">
       <img src="https://codecov.io/gh/anuraghazra/github-readme-stats/branch/master/graph/badge.svg" />
     </a>
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">
+    <a href="https://github.com/anuraghazra/github-readme-stas/issues">
```

Comment:
Don't think you meant to change that.

Reply:
Oops...

That's my type error <( o _o)

Thank you!